<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MtbUniConfigWindow</class>
 <widget class="QDialog" name="MtbUniConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Module name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_inputs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_inputs"/>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_outputs">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>351</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Outputs</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gl_outputs"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MtbUniConfigWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MtbUniConfigWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
